include/odp/api/spec/traffic_mngr.h

```diff
@@ -584,6 +614,41 @@ void odp_tm_egress_init(odp_tm_egress_t *egress);
 int odp_tm_capabilities(odp_tm_capabilities_t capabilities[],
 			uint32_t              capabilities_size);
 
+/** Query All TM Capabilities specific to an egress
+ *
+ * The odp_tm_egress_capabilities() function can be used to obtain the complete
+ * set of TM limits supported by this implementation for a given egress.
+ * The reason that this  returns a SET of capabilities and not just one, is
+ * because it is expected  that many HW based implementations may have one set
+ * of limits for the HW and also support a SW TM implementation with a
+ * (presumably larger) different set of limits.  There are also cases where
+ * there could be more than SW implementation (one supporting say tens of
```

Comment:
Typo: "more than SW" -> "more than one SW"

Paragraph breaks would make this more readable.

There are double spaces inside sentences in a few places.

Reply:
This is still open. I think "more than SW" should be "more than one".